image/templates/helm/stackrox-secured-cluster/templates/sensor-rbac.yaml

```diff
@@ -69,7 +69,14 @@ rules:
   resources:
   - '*'
   verbs:
```

Comment:
And by the way, did you make sure SCCs indeed the only use case for the `use` verb?
Otherwise we could be taking away some niche privilege...

Reply:
You mean "mention" here in the PR or in the code as a comment?

We remove verb `use` from the `*`, because it would allow us to `use` any SCC and thus all other Roles that concern SCCs would be shadowed by this one.

Reply:
In the code as comment.

Reply:
> And by the way, did you make sure SCCs indeed the only use case for the `use` verb? Otherwise we could be taking away some niche privilege...

I am not sure how to verify whether there are any other verbs here - read the source? The docs mention only `use` everywhere when it comes to consuming the SCC by a pod. I guess `get`, `create`, `delete` could also have a meaning in this context, but whatever gets created won't be able to be used.

Reply:
My approach would be:
- see if this verb exists in vanilla k8s. SCCs are an openshift-only feature.
- if they do, ask in #sig-api-machinery on kubernetes slack perhaps, if the upstream docs don't mention it?

Reply:
I'm not sure if `use` has any use outside of PSPs and SCCs, but I am pretty sure that we do not rely on any `use` privileges in the sensor code.

Reply:
- Comment added in the code 
- The verb `use` appears in the vanilla k8s in context of PodSecurityPolicies ([example](https://docs.giantswarm.io/getting-started/rbac-and-psp/)). They come however first in `k8s.io/api/policy/v1beta1` 
- As far as I can see in the k8s API code, the verbs are just strings and an API is allowed to define custom set of verbs.

So, if we want to disallow `use` for `SCC`s only, then we would need to additionally restrict the resources, so that `use` is allowed for all but `SCC`.
We could replace the `'*`` for `resources with a list of all resources, e.g., from [here](https://kubernetes.io/docs/reference/kubectl/#resource-types). This may be a lot of entries, as OpenShift will add their own types.
Alternatively, we could allow both `*-resource` and `*-verb` on selected api groups, but we would end up listing all of them but one as we cannot allow `*` minus one particular item.

What do you think?